docs/client/rest/rest_api.md

```diff
@@ -457,13 +457,14 @@ Delete the specified engine.
 
 #### Request Parameters
 
-| Name                    | Description                   | Type             |
-|:------------------------|:------------------------------|:-----------------|
-| type                    | the engine type               | String(optional) |
-| sharelevel              | the engine share level        | String(optional) |
-| subdomain               | the engine subdomain          | String(optional) |
-| proxyUser               | the proxy user to impersonate | String(optional) |
-| hive.server2.proxy.user | the proxy user to impersonate | String(optional) |
+| Name                    | Description                         | Type              |
+|:------------------------|:------------------------------------|:------------------|
+| type                    | the engine type                     | String(optional)  |
+| sharelevel              | the engine share level              | String(optional)  |
+| subdomain               | the engine subdomain                | String(optional)  |
+| proxyUser               | the proxy user to impersonate       | String(optional)  |
+| hive.server2.proxy.user | the proxy user to impersonate       | String(optional)  |
+| kill                    | whether to kill the engine forcibly | Boolean(optional) |
```

Comment:
let's mention the default value is false

Reply:
Got it.